client/Cargo.toml

```diff
@@ -45,7 +45,9 @@ futures-util = "0.3.25"
 [dev-dependencies]
 iroha = { path = "../cli", features = ["dev-telemetry", "telemetry"] }
 iroha_wasm_builder = { version = "=2.0.0-pre-rc.13", path = "../wasm_builder" }
+iroha_genesis = { version = "=2.0.0-pre-rc.13", path = "../genesis" }
```

Comment:
this should be pulled from `dependencies` section automatically?

Reply:
what do you mean?

Reply:
it's duplicated, you can remove it